### PR TITLE
Reduce the frequency in which ~/.vault-token template is rendered

### PR DESCRIPTION
**Pull request by lawliet89**

### CODE REVIEW

File: modules/core/packer/roles/install-consul-template/files/run-consul-template.sh [hunk 1]
```diff
@@ -307,6 +307,12 @@ template {
   create_dest_dirs = true
   error_on_missing_key = true
   perms = 0600
+
+  # Vault Tokens should not change often. We can set this to something pretty long.
```
Comment (chrissng) [inline modules/core/packer/roles/install-consul-template/files/run-consul-template.sh]: @guangie88 Gotta update this too https://github.com/GovTechSG/amazon-linux-consul/blob/master/packer/roles/install-consul-template/files/run-consul-template.sh#L318

Comment (chrissng) [reply modules/core/packer/roles/install-consul-template/files/run-consul-template.sh]: Please see also https://github.com/GovTechSG/amazon-linux-consul/pull/3

**Comment by chrissng** [comment]
Please see also https://github.com/GovTechSG/amazon-linux-consul/pull/3

**Comment by guangie88** [comment]
Previously without the `wait` stanza, when does it refresh?

https://github.com/hashicorp/consul-template#configuration-file-format suggests that there was no default value for `wait { min }`, unless there was one set globally?

**Pull request merged**